.storybook/foundations/corner-rounding/corner-rounding.mdx

```diff
@@ -0,0 +1,45 @@
+import { Meta, Canvas, Story, Title } from '@storybook/blocks';
+import * as CornerRoundingStories from './corner-rounding.stories.js';
+import ComponentSizeTokensSmall from './component-size-tokens-small.mdx';
+import ComponentSizeTokensMedium from './component-size-tokens-medium.mdx';
+import * as CheckboxCornerRoundingStories from './checkbox-corner-rounding.stories.js';
+import * as ActionButtonCornerRoundingStories from './action-button-corner-rounding.stories.js';
+
+<Meta of={CornerRoundingStories} />
+
+<Title />
+
+## Alias tokens
+
+<Story of={CornerRoundingStories.CornerRounding} />
+
+## Component size tokens
+
+In addition to the alias tokens listed above, Spectrum 2 also provides component size alias tokens that align with global size tokens:
```

Comment:
I have some similar thoughts on simplifying the table displayed here as well. I think the table should only have two columns, "Component Size Token" and "Value". I'm a little unclear about the purpose of the "Alias Token (if available)" column here.

Reply:
My thought in having the tables was that if I were building (or I guess even migrating) a component that required corner-rounding, for instance a checkbox, I'd want to know that there were all these nice component size tokens available to use on each size variant of the checkbox.

Since we're moving toward medium size being the default, I'd want to know that I probably should use the alias token `--spectrum-corner-radius-small-default` for corner-rounding on the default checkbox, but then use the component size tokens like `--spectrum-corner-radius-small-size-large` for the large size checkbox, and other component size tokens for other sizes.

Given that, maybe some sort of note on using the alias token for default sizes would be nice here?

We didn't get a specific ask from design to show all the non-alias tokens, but the design spec does line up all the different tokens (alias, component size, global) with the values so you can see where they fall, and that seemed like it might be a nice thing to try to capture here, but I can also see how that might be unnecessary--it doesn't seem like we typically include that level of detail in Storybook.

Does that help either of you lean one way or another about whether to include the tables/what to include on them?

Reply:
Thanks for this excellent explanation. I was confused about the use of non-alias tokens, but since they'll be used in some cases we definitely want to surface them here as you've done 👏 

Like you suggested, I'm going to add a note to use alias tokens when possible.